PLAYER/SetPlayerTeam.md

```diff
@@ -8,11 +8,8 @@ ns: PLAYER
 void SET_PLAYER_TEAM(Player player, int team);
 ```
 
-```
-Set player team on deathmatch and last team standing..  
-```
+Set the player current team.
 
 ## Parameters
-* **player**: 
-* **team**: 
-
+* **player**: The player index, or -1 to get the local player ped.
```

Comment:
Usage of `-1` is discouraged, usually `PlayerPedId()` is used in Lua in favor of it.